services/src/handlers/wfs.rs

```diff
@@ -239,20 +241,31 @@ fn get_capabilities(_request: &GetCapabilities) -> Result<HttpResponse> {
         </FeatureType>       
     </FeatureTypeList>
 </wfs:WFS_Capabilities>"#,
-        wfs_url = wfs_url
+        wfs_url = wfs_url,
+        workflow = workflow
     );
 
     Ok(HttpResponse::Ok()
         .content_type(mime::TEXT_HTML_UTF_8)
         .body(mock))
 }
 
+fn wfs_url(workflow: WorkflowId) -> Result<Url> {
+    let base = crate::util::config::get_config_element::<crate::util::config::Web>()?
+        .external_address
+        .ok_or(Error::ExternalAddressNotConfigured)?;
+
+    base.join("/wfs/")?
+        .join(&workflow.to_string())
+        .map_err(Into::into)
+}
+
 /// Retrieves feature data objects.
 ///
 ///  # Example
 ///
 /// ```text
-/// GET /wfs?request=GetFeature&version=2.0.0&typeNames=test&bbox=1,2,3,4
+/// GET /wfs/93d6785e-5eea-4e0e-8074-e7f78733d988?request=GetFeature&version=2.0.0&typeNames=registry:93d6785e-5eea-4e0e-8074-e7f78733d988&bbox=1,2,3,4
```

Comment:
Isn't this incompatible with the `GetCapabilities` since it only contains `93d6785e-5eea-4e0e-8074-e7f78733d988` and not `registry:93d6785e-5eea-4e0e-8074-e7f78733d988`?

Reply:
I don't know. maybe we should just omit the namespace?! We only support `registry` now anyway. We don't use a namespace in the other OGC services, but I am not sure whether it's allowed to omit it in WFS.

Reply:
Haven't seen at this point that this is a namespace. Can we introduce this namespace everywhere? Maybe this is beneficial for the future if we have remote registries :shrug:.

Reply:
I removed the namespace and adjusted python and wave 27837685c2fd299223f210173a7c54e195b12342